src/Styleguide/Components/__tests__/ArtistCard.test.tsx

```diff
@@ -18,39 +19,39 @@ describe("ArtistCard", () => {
       name: "Francesca DiMattio",
       formatted_nationality_and_birthday: "American, b. 1979",
       id: "percy",
+      " $fragmentRefs": null,
+      " $refType": null,
     },
   }
 
   beforeAll(() => {
-    window.matchMedia = undefined // Immediately set matching media query in Boot
+    window.matchMedia = undefined // Immediately set matching media query in MockBoot
   })
 
   it("is responsive", () => {
     const small = mount(
-      <Boot initialMatchingMediaQueries={["xs"]}>
+      <MockBoot breakpoint="xs">
```

Comment:
`breakpoint` is a bit easier on the hands. At some point if we wanted to make it more correct I think we could add a `mediaQuery` prop and merge the two together as it's passed into Boot.